SQF/dayz_code/Configs/CfgLoot/Groups/Zombies/Worker.hpp

```diff
@@ -8,12 +8,12 @@ ZombieWorker[] =
 	{Loot_MAGAZINE,		2,		ItemDocument},
 	{Loot_MAGAZINE,		2,		ItemWire},
 	{Loot_MAGAZINE,		3,		ItemTankTrap},
-	{Loot_MAGAZINE,		3,		ItemComboLock},
+	{Loot_MAGAZINE,		2,		ItemComboLock},
 	{Loot_MAGAZINE,		2,		ItemSledgeHead}
 };
 
 ZombieWorkerViral[] =
 {
-	{Loot_GROUP,		10,		ZombieCivilian},
+	{Loot_GROUP,		10,		ZombieWorker},
```

Comment:
Should be `ZombieWorkerViral`. Also the Suit one above should be `ZombieSuitViral`